front/components/assistant_builder/AssistantBuilder.tsx

```diff
@@ -923,17 +933,20 @@ export default function AssistantBuilder({
               </div>
             </div>
           </div>
-          <TeamSharingSection
-            owner={owner}
-            initialScope={initialBuilderState?.scope ?? defaultScope}
-            newScope={builderState.scope}
-            setNewScope={(
-              scope: Exclude<AgentConfigurationScope, "global">
-            ) => {
-              setEdited(true);
-              setBuilderState((state) => ({ ...state, scope }));
-            }}
-          />
+          {agentConfigurationId && (
```

Comment:
why remove it? We want to have TeamSharing on agent creation?

Reply:
You should allow agentConfigurationId = null as we def want team sharing to be visible for new agents